packages/hoprd/src/index.ts

```diff
@@ -226,10 +228,14 @@ async function main() {
       })
     }
 
-    logs.log('node is waiting for funds to', (await node.getEthereumAddress()).toHex())
+    const ethAddr = (await node.getEthereumAddress()).toHex()
+    // 0.1 NativeBalance
+    const fundsReq = new NativeBalance(new BN('100000000000000000')).toFormattedString()
```

Comment:
Two comments here
- If we are using a specific value as a minimum for our node, it should be stored in our `hopr-utils` as a constant, e.g. `const fundsReq = MINIMAL_NODE_FUNDS`, so we don't use magic numbers and have to chase this in the future.
- Ideally, if we are using `bn.js` for a single purpose value, we can probably pre-build that value in `hopr-utils` and just import the generated `string` rather than embedding `hoprd` with a new dependency, even if it's a small one

Reply:
do we have the constants somewhere? I looked it up and didn't find any, I would prefer to fix it now

Reply:
found it, must have missed it before, fixing